packages/icons/umd.dependencies.json

```diff
@@ -0,0 +1,8 @@
+{
+  "react": {
+    "name": "react",
+    "var": "React",
+    "version": "16.13.1",
+    "path": "/umd/react.development.js"
+  }
+}
```

Comment:
Let's add dependencies.json in gitignore

Reply:
no this is the template used to generate it in the dist folder which is ignored